.github/workflows/feature-ci.yml

```
@@ -38,12 +38,12 @@ jobs:
       - name: Grant execute permission for Gradlew
         run: chmod +x ./gradlew
 
-      - name: Start Redis
-        uses: supercharge/redis-github-action@1.7.0
-        with:
-          redis-version: ${{ secrets.REDIS_TEST_VERSION }}
-#          redis-remove-container: true
-          redis-password: ${{ secrets.REDIS_TEST_PASSWORD }}
+#      - name: Start Redis
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)